app/templates/src/static/css/brand-palette.less

```diff
@@ -3,28 +3,67 @@
    Color variables
    ========================================================================== */
 
-// Official Palette as of October 29, 2013 (Brand 2.0)
+// Official Palette current as of April 9, 2015.
```

Comment:
Fixed in https://github.com/anselmbradford/generator-cf/commit/b81720f784a2e945ef39dbd3eefc7cc174376e89